service.subtitles.subsceneplus/service.py

```diff
@@ -0,0 +1,304 @@
+# -*- coding: utf-8 -*- 
+
+import os
+import sys
+import xbmc
+import urllib
+import xbmcvfs
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+import shutil
+import unicodedata
+import time
+from difflib import SequenceMatcher
+
+ADD_ON = xbmcaddon.Addon()
+AUTHOR = ADD_ON.getAddonInfo('author')
+SCRIPT_ID = ADD_ON.getAddonInfo('id')
+SCRIPT_NAME = ADD_ON.getAddonInfo('name').encode('utf-8')
+VERSION = ADD_ON.getAddonInfo('version')
+
+CWD = xbmc.translatePath(ADD_ON.getAddonInfo('path')).encode('utf-8')
+PROFILE = xbmc.translatePath(ADD_ON.getAddonInfo('profile')).encode('utf-8')
+RESOURCE = xbmc.translatePath(os.path.join(CWD, 'resources', 'lib')).encode('utf-8')
+TEMP = xbmc.translatePath(os.path.join(PROFILE, 'temp', '')).encode('utf-8')
+
+sys.path.append(RESOURCE)
```

Comment:
please avoid using sys.path.append and import your file instead:

`from resources.lib.foo import bar`